r/version_5/functions/save_spss_qs_v5.R

```diff
@@ -11,6 +11,7 @@ save_spss_qs <- function(spss_file, qs_file, country, keep){
   ## Save as an rds file
   names(dt_) <- tolower(names(dt_))
   dt_ <- dt_[substr(cultureinfo,4,5)==keep]
+  if (nrow(dt_) == 0) stop(paste("Empty data table: ", spss_file, group, keep))
```

Comment:
Why would it be entirely empty?

Reply:
it is helpful in trouble shooting - i can't remember what happened but I made an error in the spss_file and it took a long time to figure out what went wrong

Reply:
Thanks